homeassistant/components/vallox/sensor.py

```diff
@@ -140,20 +150,40 @@ class ValloxSensorEntityDescription(SensorEntityDescription):
         icon="mdi:fan",
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=PERCENTAGE,
-        sensor_type=ValloxFanSpeedSensor,
+        entity_type=ValloxFanSpeedSensor,
+    ),
+    ValloxSensorEntityDescription(
```

Comment:
Should this entity have an entity category?

Reply:
We have "diagnostic" everywhere in ValloxSensorEntity class.
_attr_entity_category = EntityCategory.DIAGNOSTIC